src/main/webapp/index.html

```diff
@@ -41,65 +41,65 @@ <h2 class="panel-row__header u-align-with-first-panel u-text-shadow">Services (a
       <!-- Gmail container -->
       <div class="panel u-border-mail">
         <div class="panel__header">
-          <div class="panel__content-entry u-remove-top-margin">
+          <div class="panel__content-entry u-remove-top-margin u-vertical-center">
```

Comment:
At a glance, it looks like `u-vertical-center` is always a class alongside `panel__content-entry`. Unless there is a case where `panel__content-entry` is used without `u-vertical-center`, would it be possible to merge these two together?

Reply:
Agreed - I'm getting rid of this utility. I initially thought it would be helpful, but it's actually just redundant.